### DISCUSSION TITLE
20220627(월) Sprint2 계획

### DISCUSSION THREAD

**Discussion by Likilee** [innovationacademy-kr/42dashboard#68] [Sprint]
### Attending

- 42Dashboard 팀 전원(멘토님 제외 ㅜㅜ)

#### COMMON

-  유스케이스 예시 루나님한테 받기 (기존 보고서 자료 데이터 지우고 보내달라고 요청)

#### FE TEAM

- 데이터 FETCHING API 부분 제외 구현 완료

#### BE TEAM

- ####  로그인 기능 구현

- Updater 구현 완료
- graphql 쿼리, 뮤테이션 정리해서 API 명세 전달.

### Action Item

- [ ] 스프린트 태스트 예상 작업 기간 작성하기
- [ ] 작업을 이번 스프린트 보다 더 구체적이고 작게 만들기
- [ ] 스크럼 때, 프로젝트 보드를 보면서 진행하기
- [ ] 프로젝트 보드에 이슈에 관련 학습 기록(디스커션 ID), 관련 이슈ID, PULL REQUEST 멘션 잘해 주자

**Comment by soaple** [comment]
수요일에 만나요ㅎㅎ
